plugins/hifiLeapMotion/CMakeLists.txt

```diff
@@ -6,7 +6,12 @@
 #  See the accompanying file LICENSE or http:#www.apache.org/licenses/LICENSE-2.0.html
 #
 
-find_package(LEAPMOTION)
+if (USE_LEAPMOTION)
+    find_package(LEAPMOTION)
+else()
+    message(STATUS "Use of Leap Motion SDK not enabled. Run cmake with -DUSE_LEAPMOTION=1 to use it.")
```

Comment:
```suggestion
    message(STATUS "Use of Leap Motion SDK not enabled. Run CMake with -DUSE_LEAPMOTION=1 to use it.")
```

Reply:
The command itself is lowercase though

Reply:
Ah, in that context then nevermind! I always go around fixing the way CMake is typed as its name is "CMake" and not "cmake", heh.